inst/rmarkdown/templates/science/resources/doubts.md

```diff
@@ -0,0 +1,5 @@
+1. Need a help to figure out how to create a pandoc template for the author's section
+2. Is there a way to get Pandoc to render all images at the end? 
```

Comment:
The journal requirements for figures are as follows - 

> If the manuscript is an initial submission, you should include figures within the PDF file that you upload, preferably at the end of the text. 
> For the final submission (after peer review)- Save your figures as separate files, using Science's guidelines for preparing figures, and upload the figures separately with figure captions at the end of your LaTeX file as ordinary text (i.e., do not use the {figure} environment)
> - [Science - Authoring articles with LaTeX](https://www.sciencemag.org/authors/preparing-manuscripts-using-latex)

So, my thought was to let the author set a variable to either "Draft" or "Final" in the YAML header and then compile the document as needed without having to change the primary rmarkdown file.

Reply:
This is a good thought. I hope this will be possible to do so. You may try. I think it will need to be in the R function that would set or unset a value of a **knitr** option. It could also be a variable set in the setup chunk to use in chunks with figures. This variable would modify the value of the chunk option accordingly. I let you try what you think is better. 

But Pandoc itself won't be able to do that (unless maybe using a Lua filter, but it seems too much)